spec/jsonapi_rails4/middleware_spec.rb

```diff
@@ -116,11 +106,24 @@
         'data' => {
           'type' => 'users',
           'attributes' => {
-            'email' => 'example.com',
+            'email' => 'john.doe@example.com',
             'password' => '12345678'
           }
         }
       )
     end
   end
+
+  context 'content body is empty' do
+    let(:env) do
+      Rack::MockRequest.env_for('',
+        'rack.input' => StringIO.new({}.to_json),
```

Comment:
here content body is not empty and contains '{}'
i'd like also to test fully empty body StringIO.new('')